tests/spec/core/dfn-spec.js

```diff
@@ -153,4 +153,75 @@ describe("Core — Definitions", () => {
       expect(el.hash).toBe("#dfn-first");
     }
   });
+
+  describe("internal slot definitions", () => {
+    const body = `
+      <section data-dfn-for="Test" data-cite="HTML">
+        <h2>Internal slots</h2>
+        <pre class="idl">
+          [Exposed=Window]
+          interface Foo{};
+        </pre>
+        <p>
+          <dfn id="attribute">
+            [[\\internal slot]]
+          </dfn>
+          <dfn id="method">
+            [[\\I am_a method]](I, really, ...am)
+          </dfn>
+          <dfn id="parent" data-dfn-for="Window">[[\\internal slot]]</dfn>
+          <dfn id="broken" data-dfn-for="">[[\\broken]]</dfn>
+        </p>
+        <section data-dfn-for="">
+          <h2>.</h2>
+          <p>
+            <dfn id="broken-parent">[[\\broken]]</dfn>
+          </p>
+        </section>
+        <p>
+        {{Test/[[internal slot]]}}
+        {{Test/[[I am_a method]](I, really, ...am)}}
+        {{Window/[[internal slot]]}}
+        </p>
+      </section>
+    `;
+
+    it("sets the data-dfn-type as an attribute", async () => {
+      const ops = makeStandardOps(null, body);
+      const doc = await makeRSDoc(ops);
```

Comment:
This is repeated (with same args) in each test here. Can probably move it into a `beforeAll`?

Reply:
It's specific to this particular `describe()` group, so beforeAll is probably not ideal... wish they had a "before you run these tests".

Reply:
I think `beforeAll` is scoped to `desribe()` blocks

Reply:
Ok, will try. The docs said they beforeAll was global.